scripts/generator/graphql_generator/csharp/Cart.cs.erb

```diff
@@ -170,17 +170,38 @@ namespace <%= namespace %> {
             }
         }
 
-        public void SetShippingAddressAndEmail(MailingAddressInput mailingAddressInput, string email, CompletionCallback callback) {
+        public void SetEmailAddress(string email, CompletionCallback callback) {
```

Comment:
I wonder if we should still have a method that does `SetShippingAddressAndEmail`. I don't think this is crucial right now but it's something we might want to do.